doc/manual/parallel-computing.rst

```diff
@@ -880,22 +882,22 @@ times during the worker's lifetime with appropriate ``op`` values:
 - with ``:finalize`` for cleanup purposes.
 
 
-Cluster Managers with custom transports
+Cluster Managers with Custom Transports
 ---------------------------------------
 
 Replacing the default TCP/IP all-to-all socket connections with a custom transport layer is a little more involved.
 Each Julia process has as many communication tasks as the workers it is connected to. For example, consider a Julia cluster of
-32 processes in a all-to-all mesh network:
+32 processes in an all-to-all mesh network:
 
-- Each Julia process thus has 31 communication tasks
-- Each task handles all incoming messages from a single remote worker in a message processing loop
-- The message processing loop waits on an :class:`IO` object (for example, a :class:`TCPSocket` in the default implementation), reads an entire
-  message, processes it and waits for the next one
-- Sending messages to a process is done directly from any Julia task - not just communication tasks - again, via the appropriate
-  :class:`IO` object
+- Each Julia process thus has 31 communication tasks.
+- Each task handles all incoming messages from a single remote worker in a message-processing loop.
+- The message-processing loop waits on an :class:`IO` object (for example, a :class:`TCPSocket` in the default implementation), reads an entire
+  message, processes it and waits for the next one.
+- Sending messages to a process is done directly from any Julia task---not just communication tasks---again, via the appropriate
```

Comment:
Putting Unicode in the docs wreaks havoc with the LaTeX/PDF generator, thanks to sphinx being hard-coded to use only pdflatex.